pkg/git/git.go

```diff
@@ -97,6 +120,21 @@ func (git Client) DeleteLocalRepo() error {
 	return fmt.Errorf("no cloned directory to delete")
 }
 
+// RepoName returns the name of the repository given its URL
+func RepoName(repoURL string) (string, error) {
```

Comment:
It looks like a function which is complex enough to have unit tests?

Reply:
added unit tests